components/listings/filter-sidebar.tsx

```
@@ -0,0 +1,301 @@
+        </div>
+      )}
+
+      <Accordion type="multiple" defaultValue={["category", "price", "location"]}>
+        <AccordionItem value="category">
+          <AccordionTrigger>Category</AccordionTrigger>
+          <AccordionContent>
+            <div className="space-y-2">
+              <div
+                className={`px-2 py-1 rounded cursor-pointer ${selectedCategory === "all" ? "bg-primary text-primary-foreground" : "hover:bg-muted"}`}
+                onClick={() => setSelectedCategory("all")}
+              >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)